### ISSUE TITLE
Create RPC manifest

### ISSUE THREAD

**Issue by mycognosist** [Kuska-ssb/kuska-ssb#28]
It seems useful at this point to have a list of RPCs supported by kuska (separated by request and response).

I intend on generating the list and including the shape in which data is expected for each method.

**Comment by decentral1se** [comment]
Might be useful to see that `go-ssb` simply provides a JSON response which is hard-coded:

> https://github.com/ssbc/go-ssb/blob/3d0822793d8ba3c2e2166e0f58594a048860052e/sbot/manifest.go#L60-L149

**Comment by mycognosist** [comment]
Yeah maybe a simple `manifest.json` is a reasonable approach for the time being.
